frontend/app/.server/routes/mappers/benefit-renewal.state.mapper.ts

```diff
@@ -194,8 +178,7 @@ export class DefaultBenefitRenewalStateMapper implements BenefitRenewalStateMapp
       demographicSurvey,
       dentalBenefits: this.toDentalBenefits({
         existingDentalBenefits: clientApplication.dentalBenefits,
-        hasFederalBenefitsChanged,
-        hasProvincialTerritorialBenefitsChanged,
+        hasFederalProvincialTerritorialBenefitsChanged: !!dentalBenefits,
```

Comment:
The renew adult/child state should have the result of `hasFederalProvincialTerritorialBenefitsChanged` based on the user's response.

We should be adding `hasFederalProvincialTerritorialBenefitsChanged` to the `RenewAdultChildState` interface and using that result here instead.

(same thing for mapping `changeIndicaors` > `hasFederalProvincialTerritorialBenefitsChanged` above)